link_shortener/templates/all_links.html

```diff
@@ -30,7 +30,7 @@
             </a>
           </td>
           <td>
-            <a href="/delete/{{ row.id }}">
+            <a href="/delete/confirm/{{ row.id }}">
```

Comment:
Mám pocit, že de facto standard na tohle je /delete/id/confirm. Ale to jen pro zvážení kdyby se ti to líbilo víc.

Reply:
Standardy radi, dikes. :)

Reply:
@view_blueprint.route('/delete/<link_id>/confirm/', methods=['GET'])